### ISSUE TITLE
Add alt text to images

### ISSUE THREAD

**Issue by peterwoodworth** [aws-samples/aws-cdk-intro-workshop#648]
### General Information

- [X] I may be able to implement this feature request
- [ ] This feature might incur a breaking change

### Description

Images should have alt text for accessibility

### Proposed Solution

Add alt text to images

**Comment by peterwoodworth** [comment]
To be handled after #636 is merged
